evalit/_base.py

```diff
@@ -76,29 +115,68 @@ def __init__(
             config = self.load_yaml(config)
         if not isinstance(config, dict):
             raise TypeError(
-                f"Invalid type for config. Expected Dict[str, str]. Got {type(config)}"
+                (
+                    f"Invalid type for config. Expected Dict[str, str]. "
+                    f"Got {type(config)}"
+                )
             )
         self.config = copy.deepcopy(config)
         if debug:
-            logger.debug(self.__get_redacted_cfg())
+            logger.debug(self._get_redacted_cfg(self.config))
```

Comment:
`self._get_redacted_cfg` is an object method, why the need to send in `self.config` object variable?

Reply:
The `_get_redacted_cfg` is actually a static method. I think the way I am accessing here is by creating the illusion that it's an object method. But since static methods can be accessed using objects also, so...